### ISSUE TITLE
provide windows bindings

### ISSUE THREAD

**Issue by holgerbrandl** [holgerbrandl/send2terminal#7]
see supported packages under https://github.com/randy3k/SendCode

**Comment by holgerbrandl** [comment]
Also see https://github.com/randy3k/Terminus

**Comment by holgerbrandl** [comment]
From
https://github.com/randy3k/SendCode/blob/383814a3635bb937b5412adfdbd4c1c5f98d271c/code_sender/conemu/__init__.py#L76 this seems to work

```

C:\Users\Holger>C:\Users\Holger\bin\cmder\vendor\conemu-maximus5\ConEmu\ConEmuC.exe  -GuiMacro:0 Paste(2,"ls")
OK

C:\Users\Holger>C:\Users\Holger\bin\cmder\vendor\conemu-maximus5\ConEmu\ConEmuC.exe -GuiMacro:0 Keys("Return")

```

**Comment by holgerbrandl** [comment]
Fixed in v1.4
